src/test/java/com/synopsys/integration/blackduck/comprehensive/InstallAndRunSignatureScannerTestIT.java

```diff
@@ -73,7 +73,7 @@ public void cleanUp() {
         FileUtils.deleteQuietly(scannerDirectoryPath);
     }
 
-//    @Test
+    @Test
```

Comment:
Bringing back this test now that certificates are fixed on our butler instance and comprehensive pipeline is green again